spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -99,6 +101,91 @@ private SpoonRunner(String title, File androidSdk, File applicationApk, File ins
     }
   }
 
+  /** Returns {@code false} if a test failed on any device. */
+  static boolean parseOverallSuccess(SpoonSummary summary) {
```

Comment:
I just ran the intellij formatter on this class with the SquareAndroid code style selected and it didn't re-order any methods.  That said, it's possible this is a thing with specific IntelliJ versions (or maybe i have an old version of the code style)...can you just manually move the methods back?  I want to try and keep the git blame readable on this stuff so it's easier to figure out the intention behind the various changes.